src/main/java/com/harrison/messaging/Consumer.java

```diff
@@ -0,0 +1,25 @@
+package com.harrison.messaging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("local")
+public class Consumer {
+    
+    private static final Logger LOGGER = LoggerFactory.getLogger(Consumer.class);
+    
+    private int count = 1;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "count" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AS1068)